lang/testfile

```diff
@@ -0,0 +1,2 @@
+This creates a change that should trigger a new build
```

Comment:
remove these?

Reply:
or are you wanting to merge and then remove them to test something?

Reply:
ah, yeah they should be gone
thanks